filebeat/harvester/log.go

```diff
@@ -186,52 +216,38 @@ func (h *Harvester) openFile() error {
 		return err
 	}
 
-	// yay, open file
-	h.file = source.File{f}
 	return nil
 }
 
 func (h *Harvester) initFileOffset(file *os.File) error {
```

Comment:
@urso After your input I decided to rewrite / cleanup this part here. I think it is identical to before, but now max 1 Seek instead of 2 are needed and it seems easier to read. Can you have a close look if it looks sane for you?